site/documentation/content/getting-started/_index.md

```diff
@@ -120,7 +120,7 @@ export REGISTRY_IP=hono.eclipseprojects.io
 export HTTP_ADAPTER_IP=hono.eclipseprojects.io
 export MQTT_ADAPTER_IP=hono.eclipseprojects.io
 export KAFKA_IP=hono.eclipseprojects.io
-export APP_OPTIONS="-H hono.eclipseprojects.io -P 9094 --ca-file /etc/ssl/certs/ca-certificates.crt -u hono -p hono-secret"
+export APP_OPTIONS="--sandbox --ca-file /etc/ssl/certs/ca-certificates.crt"
```

Comment:
The `sandbox` parameter is also mentioned/used in these files:
- `examples/quickstart-python/README.md`
- `examples/quickstart-python/quickstart.py`
- `site/documentation/content/user-guide/amqp-adapter.md`

The `--ca-file` parameter should be added there as well.

Reply:
Thanks for spotting :+1:

Reply:
In addition to the first sandbox CLI command in
`site/documentation/content/user-guide/amqp-adapter.md`
I think it would be good also to add the `--ca-file` parameter to the remaining CLI commands, possibly as `$CA_FILE_PARAM` to keep the whole command shorter.

Reply:
I don't think that the length of the command is much of a problem as it will be copied anyway ...